hal.cabal

```diff
@@ -131,7 +131,7 @@ test-suite hal-test
       ScopedTypeVariables
   ghc-options: -Wall -Wcompat -Wincomplete-record-updates -Wincomplete-uni-patterns -Wredundant-constraints -fno-warn-partial-type-signatures -fno-warn-name-shadowing -fwarn-tabs -fwarn-unused-imports -fwarn-missing-signatures -fwarn-incomplete-patterns -threaded -rtsopts -with-rtsopts=-N
   build-depends:
-      aeson >=1.2.0.0 && <1.6 || >=2.0.0.0 && <2.2.0.0
+      aeson >=1.2.0.0 && <1.6 || >=2.0.0.0 && <2.2.1.0
```

Comment:
I think this should be 2.3.0.0 — PVP says that no breaking changes can come in from later 2.2.x.x releases.